tests/e2e/scripts/prow-utilities.sh

```diff
@@ -294,7 +295,11 @@ function download_client_binaries() {
 
 function get_apiproxy_service() {
   if [[ "${1}" == "bookstore" ]]; then
-    echo "bookstore.endpoints.cloudesf-testing.cloud.goog"
+    if [ -n "${2}" ]; then
+      echo "bookstore-using-sa-cred.endpoints.cloudesf-testing.cloud.goog"
```

Comment:
Why do we need a different Endpoints Service for this? We should be able to re-use the bookstore one, right?

Conceptually, deploying with or without the SA does not require any difference in the service. So our tests should follow that.

Reply:
It is unrelated by SA.

The reason is that two tests will deploy the same http bookstore service in the same test causing one deployment fail.